From 2e731ec279bf35254494610c402417df98d1914f Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Sat, 1 Apr 2017 16:52:13 +0800
Subject: [PATCH] arm: dts: rk3288-android: fix rga to rockchip,rga2

Fix compatible of rga to "rockchip,rga2", and modify the clock names
to work fine with current driver.

Change-Id: Idfdae4f3bc2218472261546024b331c33bd3c270
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-android.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-android.dtsi b/arch/arm/boot/dts/rk3288-android.dtsi
index a6f088f36d39..b7463a032959 100644
--- a/arch/arm/boot/dts/rk3288-android.dtsi
+++ b/arch/arm/boot/dts/rk3288-android.dtsi
@@ -184,6 +184,12 @@
 	};
 };
 
+&rga {
+	compatible = "rockchip,rga2";
+	clocks = <&cru ACLK_RGA>, <&cru HCLK_RGA>, <&cru SCLK_RGA>;
+	clock-names = "aclk_rga", "hclk_rga", "clk_rga";
+};
+
 &usb_otg {
 	compatible = "rockchip,rk3288_usb20_otg";
 	clocks = <&usbphy0>, <&cru HCLK_OTG0>;
-- 
2.35.3

